lib/stm32/common/ltdc_common_f47.c

```diff
@@ -67,20 +67,25 @@ void ltdc_set_tft_sync_timings(uint16_t sync_width,    uint16_t sync_height,
 	LTDC_TWCR = (w << 16) | (h << 0);
 }
 
-void ltdc_setup_windowing(uint8_t  layer_number,
-			  uint16_t h_back_porch, uint16_t v_back_porch,
-			  uint16_t active_width, uint16_t active_height)
-{
-	active_width  += h_back_porch - 1;
-	active_height += v_back_porch - 1;
-	/*assert((h_back_porch & 0xfff == h_back_porch) &&
-		 (v_back_porch  & 0xfff == v_back_porch) &&
-		 (active_width & 0xfff == active_width) &&
-		 (active_height & 0xfff == active_height));*/
-	LTDC_LxWHPCR(layer_number) = (active_width  << 16) |
-				     (h_back_porch << 0);
-	LTDC_LxWVPCR(layer_number) = (active_height << 16) |
-				     (v_back_porch << 0);
+/*---------------------------------------------------------------------------*/
+/** @brief LTDC Windowing Setup
+@param[in] layer_number unsigned int8. @ref ltdc_layer_num
+@param[in] h_back_porch unsigned int16. Horizontal Back Porch
+@param[in] v_back_porch unsigned int16. Vertical Back Porch
+@param[in] h_sync unsigned int16. Horizontal Sync
+@param[in] v_sync unsigned int16. Vertical Sync
```

Comment:
personally, if the comments are just the variable name, I don't really see any gain.... (applies to porch/sync vars)  (also, in general, there's no reason for the type information, that's already in the decl)

Reply:
I agree, although I followed the format in the dma documentation here even though it was vague it seems odd to omit some. Easy enough to remove these if you'd like.